# rviz_type currently accepts "tall_robot", "short_robot", "human"
string type

# The id of the agent, used to define the object localisation topic
string agent_id

# Accepts "none", "red", "green", "blue", "white", "black"
string optional_color

# The topic in shich the location will be published periodically/latched
string location_topic
string location_attachment
